sdk/monitor/monitor-query/src/metricsQueryClient.ts

```diff
@@ -123,10 +130,10 @@ export class MetricsQueryClient {
     resourceUri: string,
     options: ListMetricDefinitionsOptions = {}
   ): AsyncIterableIterator<Array<MetricDefinition>> {
-    const segmentResponse = await this._definitionsClient.metricDefinitions.list(
+    const segmentResponse = await tracingClient.withSpan("MetricsQueryClient.listSegmentOfMetricDefinitions",options, async (updatedOptions) => await this._definitionsClient.metricDefinitions.list(
```

Comment:
I'm surprised prettier didn't format this? Or maybe it did and that's why build is failing I haven't checked...